CustomConverterTest fails because the custom converters are not in the test archive

I get test failures from CustomConverterTests with stacks such as this one:

```
Caused by: java.util.ServiceConfigurationError: org.eclipse.microprofile.config.spi.Converter: Provider org.eclipse.microprofile.config.tck.converters.custom.IntegerConverter not found
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:588)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1211)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1220)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator$1.run(ServiceLoader.java:1267)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator$1.run(ServiceLoader.java:1266)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:673)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1269)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1299)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1384)
	at java.base/java.lang.Iterable.forEach(Iterable.java:74)
	at io.smallrye.config.SmallRyeConfigBuilder.discoverConverters(SmallRyeConfigBuilder.java:107)
	at io.smallrye.config.SmallRyeConfig.buildConverters(SmallRyeConfig.java:112)
	at io.smallrye.config.SmallRyeConfig.<init>(SmallRyeConfig.java:63)
```

This is because the ShrinkWrap statement adds the custom converters as service providers without adding the class file to the archive.

--- a/tck/src/main/java/org/eclipse/microprofile/config/tck/CustomConverterTest.java
+++ b/tck/src/main/java/org/eclipse/microprofile/config/tck/CustomConverterTest.java
@@ -47,7 +47,7 @@ public class CustomConverterTest extends Arquillian {
             .create(JavaArchive.class, "customConverterTest.jar")
             .addClass(CustomConverterTest.class)
             .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
-            .addAsServiceProvider(Converter.class,
+            .addAsServiceProviderAndClasses(Converter.class,
                                   IntegerConverter.class,
                                   LongConverter.class,
                                   DoubleConverter.class,

